src/renderers/shaders/ShaderChunk/shadowmap_pars_vertex.glsl.js

```diff
@@ -1,4 +1,12 @@
 export default /* glsl */`
+
+#if NUM_SPOT_LIGHT_COORDS > 0
+
+  uniform mat4 spotShadowMatrix[ NUM_SPOT_LIGHT_COORDS ];
+  varying vec4 vSpotShadowCoord[ NUM_SPOT_LIGHT_COORDS ];
+
+#endif
+
```

Comment:
This code has been moved since you can use `SpotLight` map without using shadow map, right?

Reply:
You are right, but actually map without shadow was only half-way supported ... until today's commit !
It's fresh so if anyone wants this PR merged, please test it and give me feedback !